packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -114,8 +114,8 @@ contract OPContractsManager is ISemver {
 
     // -------- Constants and Variables --------
 
-    /// @custom:semver 1.0.0-beta.26
-    string public constant version = "1.0.0-beta.26";
+    /// @custom:semver 1.0.0-beta.27
+    string public constant version = "1.0.0-beta.27";
```

Comment:
Unrelated and we can track separately, but when do we want to release 1.0.0? It could be for the holocene release (and still only supports deploys), or as part of the OPCM Upgrades release. I would advocate for the latter, it feels simpler to call that 1.0.0 since it's a bigger change a new paradigm